Watchman.DomainModel/Settings/ConfigurationItem.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Watchman.Integrations.MongoDB;
+
+namespace Watchman.DomainModel.Settings
+{
+    public class ConfigurationItem : Entity, IAggregateRoot
+    {
+        public object Value { get; set; }
```

Comment:
to nie powinno być edytowalne z zewnątrz

Reply:
```csharp
        public ConfigurationItem MapIntoBaseFormat(IMappedConfiguration mappedConfiguration)
        {
            return new ConfigurationItem
            {
                ServerId = mappedConfiguration.ServerId,
                Name = mappedConfiguration.Name,
                Value = ((dynamic)mappedConfiguration).Value
            };
        }`
chyba że przez konstruktor to przypisywać?

Reply:
tak

Reply:
ogólnie te nasze agregaty są zrobione bardzo mało agregatowo 
jest masa uproszczeń, drogi na skróty itd

ale tego że mają nie mieć {get; set;} starajmy sie trzymać